package.json

```diff
@@ -14,15 +14,16 @@
     "start": "next start -p 3000",
     "preview": "next start",
     "lint": "eslint 'src/**/*.{js,jsx,ts,tsx}'",
-    "lint:fix": "pnpm run lint -- --fix",
+    "lint:fix": "pnpm run lint --fix",
     "prettier": "prettier src --check",
-    "prettier:fix": "pnpm run prettier -- --write",
+    "prettier:fix": "pnpm run prettier --write",
     "format": "pnpm run prettier:fix && yarn run lint:fix",
     "format:check": "pnpm run prettier && yarn run lint",
     "prepare": "husky install && wonderland-crypto-husky-checks install",
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
     "test:e2e": "pnpm cypress run",
+    "test:e2e:headed": "pnpm run test:e2e --headed --no-exit",
```

Comment:
I think cypress open command already does this

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/6/commits/20ff4a20e1aefbf0a3000775fb9926e923cc5007